docs/adr/0024-cancel-upload-file-with-caption.md

```diff
@@ -0,0 +1,40 @@
+# 24. Cancel upload file with caption
+
+Date: 2024-08-19
+
+## Status
+
+Accepted
+
+- Issue: [#1972](https://github.com/linagora/twake-on-matrix/issues/1972)
+
+## Context
+
+- The caption is sent anyway, there's also a weird upload preview that disappears after you reload the page.
+
+## Decision
+1. **Update logic for cancel upload files with captions**
```

Comment:
can you use chatGPT to update the text, because it have some spelling error. for example: cancel -> cancelling